scripts/db_migrations/v2.8/convertGroupIdToLUUID.js

```diff
@@ -0,0 +1,100 @@
+// node scripts/db_migrations/v2.8/convertGroupIdToLUUID.js mongodb://username:passwordq@hostname:port/db
```

Comment:
I can't run this script: 
![image](https://user-images.githubusercontent.com/11945337/38082443-9e1cd690-333e-11e8-9e15-b649e34ced92.png)

I also had to do a npm install mongodb manually before it even did anything. It downloaded v3.0.5 and i'm guessing I think you might've ran it with a different mongodb driver?

Need more specification on which mongodb is needed, potentially add a package.json on the script v2.10 script folder. Or we need more instructions on how to run this